### ISSUE TITLE
Animation when using radio

### ISSUE THREAD

**Issue by dfr1238** [IDI-Systems/acre2#867]
**Is your feature request related to a problem?**
acre2 is great radio mod,but when seem not immersion with character,special with when using radio,and unofficial mod having fix this problem by add radio animation when talk with radio,but duo to event handler make it broken,worst way,mod didn't update for a year .
many ppl love to using and seem is good idea make it in official feature,make it more feeling like using radio in visual way

**Solution you'd like:**
-a using radio animation show in first\thrid person
-make animation can show on different stance(stand,crouch,prone) and different weapon handle,even empty hand
-have the option to enable or disable animation,and can be overwrite by server side

**Alternatives you've considered:**
-add not only one animation then can make player choose what type animation them want,or server side force witch type using

**Additional context:**
KokaKolaA3 is one who made animation mod for acre2,tho didn't update for a year now,but some screenshot can show some animation

[animation1](https://steamuserimages-a.akamaihd.net/ugc/963095016358365321/630C7E55AAA83456AB0C314ABC53636BC1C64E10/)

[animation2](https://steamuserimages-a.akamaihd.net/ugc/963095016358365053/8827A5B63D8E7A095306844954C3396BFF85A1AD/)

**Comment by jonpas** [comment]
Author of the mentioned mod was going to integrate his mod a few months ago. Not sure where it got stuck.

**Comment by dfr1238** [comment]
> 
> 
> Author of the mentioned mod was going to integrate his mod a few months ago. Not sure where it got stuck.

hope can work well,is really sweet feature

**Comment by Cyruz143** [comment]
I did a quick POC using the animations provided, it does look fairly good in-game and isn't too intrusive.

However I ran in to two issues. Firstly gestures override any animation you're currently doing, such as reloading! Because I couldn't find a reliable way to check if the player is mid reload, if you play a gesture mid-reload, it just deletes the magazine and doesn't reload the gun. Pretty much a show stopper.

Ideally you'd want to block reloads from happening when talking on the radio too as you're using one hand for comms. I was going to look in to this further but the first issue was such a show stopper I gave up...
